docs/reference/config-options.md

```diff
@@ -179,14 +179,29 @@ or, you can federate with everyone:
 ```yaml
 # federator.yaml
 optSettings:
-  setFederationStrategy:
+  federationStrategy:
     # note the 'empty' value after 'allowAll'
     allowAll:
 
 # when configuring helm charts, this becomes (note 'true' after 'allowAll')
 # inside helm_vars/wire-server:
 federator:
   optSettings:
-    setFederationStrategy:
+    federationStrategy:
       allowAll: true
 ```
+
+### Federation TLS Config
+
+When a federator connects with another federator, it does so over HTTPS. There
+are two options to configure the CA for this:
+1. `useSystemCAStore`: Boolean. If set to `True` it will use the system CA.
+1. `remoteCAStore`: Maybe Filepath. This config option can be used to specify
+   multiple certificates from either a single file (multiple PEM formatted
+   certificates concatenated) or directory (one certificate per file, file names
+   are hashes from certificate).
```

Comment:
It would be good to provide an example yaml documentation block with the exact syntax and spelling. And ti would be good to list the default.

Reply:
I've added some examples in the followup PR #1682 (commit 011e83a2fa55e10554a825b70e2f9325487b57c2), since that also contains new options.